tests/cloner_test.go

```diff
@@ -1042,14 +1042,13 @@ var _ = Describe("all clone tests", func() {
 						Expect(err).ToNot(HaveOccurred())
 					}
 
-					// All pods should be in the same node
+					// All pods should be in the same node except when the map is empty in smart clone
 					if len(podsNodeName) > 0 {
-						Expect(podsNodeName[targetDvs[0].Name]).To(Equal(podsNodeName[targetDvs[1].Name]))
-						Expect(podsNodeName[targetDvs[1].Name]).To(Equal(podsNodeName[targetDvs[2].Name]))
+						Expect(podsNodeName).To(HaveLen(1))
 					}
 				})
 
-				It("[rfe_id:1277][test_id:1899][crit:High][vendor:cnv-qe@redhat.com][level:component] Should allow multiple cloning operations in parallel for block devices", func() {
+				FIt("[rfe_id:1277][test_id:1899][crit:High][vendor:cnv-qe@redhat.com][level:component] Should allow multiple cloning operations in parallel for block devices", func() {
```

Comment:
Other than the FIts the PR looks good to me!

Reply:
whoops